src/components/subscriptions/licenses/LicenseManagementModals/LicenseManagementRemindModal.jsx

```diff
@@ -65,17 +58,18 @@ const LicenseManagementRemindModal = ({
   remindAllUsers,
   totalToRemind,
   contactEmail,
+  activeFilters,
 }) => {
   const [requestState, setRequestState, initialRequestState] = useRequestState(isOpen);
 
   const [emailTemplate, setEmailTemplate] = useState(generateEmailTemplate(contactEmail));
   const isExpired = moment().isAfter(subscription.expirationDate);
 
-  const buttonLabels = generateRemindModalSubmitLabel(remindAllUsers, usersToRemind.length, totalToRemind);
+  const buttonLabels = generateRemindModalSubmitLabel(totalToRemind);
 
-  const title = `Remind User${remindAllUsers || usersToRemind.length > 1 ? 's' : ''}`;
+  const title = `Remind User${remindAllUsers || totalToRemind > 1 ? 's' : ''}`;
 
-  const handleSubmit = () => {
+  const handleSubmit = async () => {
```

Comment:
nit: I wonder if this should use `useCallback` to avoid re-creating the function definition on every component re-render.